src/app/components/modules/progress/progress.component.ts

```diff
@@ -222,6 +222,10 @@ export class ProgressComponent extends SubscriptionsComponent implements OnInit,
 		// Define nowTime just to make things clearer
 		const nowTime = this.today.getTime();
 
+		//End of School constant created for DRY
```

Comment:
Just for consistency, we start all of our comments with a space so it'd be like this:
```
// End of School...
```